docs/api/networksocket/CellularNonIPSocket.md

```diff
@@ -0,0 +1,83 @@
+## Non-IP cellular socket
+
+The CellularNonIPSocket class provides, through standard socket send and recv member functions, the ability to send and receive Non-IP datagrams using Control Plane CIoT EPS optimisation feature of the NB-IoT networks. This feature is implemented in ControlPlane_netif class. [`ControlPlane_netif`](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_control_plane__netif.html) class.
+
+The constructor takes no parameters. To initialize the socket on a specified NetworkInterface, you must call `open` method, which takes a CellularContext pointer.
+
```

Comment:
Is this inline with the multi-homing socket operation, where they give the interface name? Could check with them

Reply:
I will update this when/if CellularNonIPSocket needs to be changed due to multi-homing feature. This reflects current implementation.